### PR TITLE
v0.0.4: spec adjustment, usability, _Entities bugfix, CI fix

### PR DESCRIPTION
**Pull request by pcarrier**
Please see individual commits.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -157,15 +156,6 @@
                     <artifactId>maven-jar-plugin</artifactId>
                     <version>${maven-jar-plugin.version}</version>
                 </plugin>
-                <plugin>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-release-plugin</artifactId>
-                    <version>${maven-release-plugin.version}</version>
```
Comment (zionts) [inline pom.xml]: Did you mean to remove this?

**Comment by zionts** [comment]
Can we add some top-level description about the changes to match spec (external is only allowed on fields), and the usability improvements around creating the directives with arguments?

**Comment by pcarrier** [comment]
@zionts No plan to have a changelog, we're not even at 0.1.x yet and all changes are backward-compatible.

**Comment by pcarrier** [comment]
Moving to 0.1.0

**Pull request closed without merging**